src/source_sampling.cpp

```diff
@@ -221,14 +223,14 @@ std::vector<double> pyne::Sampler::read_bias_pdf(moab::Range ves,
         for (j=0; j<num_e_groups; ++j){
           q_in_group += pdf[i*num_e_groups + j];
         }
-        if (q_in_group > 0) {
+        if (q_in_group > threshold) {
           for (j=0; j<num_e_groups; ++j) {
             bias_pdf[i*num_e_groups + j] =
                 volumes[i]*pdf[i*num_e_groups + j]/q_in_group;
           }
         } else {
           for (j=0; j<num_e_groups; ++j) {
-            bias_pdf[i*num_e_groups + j] = 0.0;
+            bias_pdf[i*num_e_groups + j] = pdf[i*num_e_groups + j];
```

Comment:
8000 right? The source voxel size is 20x20x20 isnt it?

On Thu, Mar 3, 2016 at 12:57 PM, Paul Wilson <notifications@github.com>
wrote:

> In src/source_sampling.cpp
> <https://github.com/pyne/pyne/pull/823#discussion_r54928471>:
>
> >            for (j=0; j<num_e_groups; ++j) {
> >              bias_pdf[i*num_e_groups + j] =
> >                  volumes[i]*pdf[i*num_e_groups + j]/q_in_group;
> >            }
> >          } else {
> >            for (j=0; j<num_e_groups; ++j) {
> > -            bias_pdf[i*num_e_groups + j] = 0.0;
> > +            bias_pdf[i*num_e_groups + j] = pdf[i*num_e_groups + j];
>
> This is a choice... I decided not, but it shouldn't really matter anyway.
> It would just result in a different biasing, although slightly higher
> because the volume is ~125.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyne/pyne/pull/823/files#r54928471>.
>